controllers/cassandrabackup_controller_test.go

```diff
@@ -185,6 +186,17 @@ var _ = Describe("CassandraBackup controller", func() {
 			fmt.Sprintf("%s:%d", getPodIpAddress(1), backupSidecarPort): {backupName},
 			fmt.Sprintf("%s:%d", getPodIpAddress(2), backupSidecarPort): {backupName},
 		}))
+
+		By("deleting the backup")
+		Expect(k8sClient.Delete(context.Background(), backup)).Should(Succeed())
```

Comment:
Thanks for the test update! 

It would be nice if possible to get coverage for a failure scenario.